clap_markdown/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I just copied this over, but I can nuke it if we think the license in the root of the directory is sufficient.